hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/snapshot/ExportSnapshot.java

```diff
@@ -328,6 +333,13 @@ private void copyFile(final Context context, final SnapshotFileInfo inputInfo,
 
         // Ensure that the output folder is there and copy the file
         createOutputPath(outputPath.getParent());
+        String family = new Path(inputInfo.getHfile()).getParent().getName();
+        String familyStoragePolicy = generateFamilyStoragePolicyKey(family);
+        if (stringIsNotEmpty(context.getConfiguration().get(familyStoragePolicy))) {
+          String key = context.getConfiguration().get(familyStoragePolicy);
+          LOG.info("Setting storage policy {} for {}", key, outputPath.getParent());
+          outputFs.setStoragePolicy(outputPath.getParent(), key);
```

Comment:
This is assumes the output path has storage policy support, is enabled, and that the storage policy in both input and output file system are compatible.

Reply:
Yes. If output fs doesn't have storage policy support, this job will fail.

And user should not set in the job configuration.

Reply:
@Apache9 here read & set those configurations